3 - PB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -3585,6 +3585,166 @@ Il rendimento positivo dello Sprint 16 è supportato dalle principali metriche e
 - EAC: l'Estimate At Completion passa dal valore 13.292,70 € (allo Sprint 15) a 12.911,54 €, considerabile come valore ottimale in quanto minore del BAC. Il costo finale del progetto va dunque a ridursi ulteriormente rispetto allo Sprint 15.
 
 
+== Sprint 17 dal 25-02-2024 al 03-03-2024
+
+=== Obiettivi raggiunti
+
+Lo Sprint 17 si è concluso raggiungendo quasi la totalità degli obiettivi pianificati, in particolare:
+
+- colloquio con Proponente in data 29/02/2024;
+
+- #glo\:
+  - aggiunte definizioni di:
+    - Jest;
+    - Virtual Private Server;
+    - Testing;
+    - Unit test;
+    - Test di integrazione;
+    - Test di sistema.
+
+- #man\:
+  - creato documento;
+  - redatta sezione Introduzione.
+
+- #ndp\:
+  - redatti i capitoli:
+    - Processo di implementazione;
+    - Processo di gestione della qualità;
+    - Processo di integrazione;
+    - Processo di verifica.
+  - normato il sistema di tracciamento rischi-risoluzioni.
+
+- #pdp\:
+  - redatto preventivo dello Sprint 16, iniziato in data 18/02/2024;
+  - redatto consuntivo dello Sprint 16, terminato in data 25/02/2024.
+
+- #pdq\:
+  - aggiornati i grafici di monitoraggio delle metriche allo Sprint 16;
+  - aggiornata la dashboard con i dati sullo Sprint 16.
+
+- #st\:
```

Comment:
la build fallisce perché non esiste in common la variabile per st, togli e metti to do. Se è già stata aggiunta la variabile con un altrà pr apri un nuovo ticket e sostituisci di nuovo con #st: per non far fallire la compilazione

Reply:
Ho fatto il merge di src in questo branch con il tasto presente in PR, quindi dovrebbe essere stato aggiornato anche common